src/github.com/cloudfoundry-incubator/webhooks/pkg/webhooks/manager/event.go

```diff
@@ -0,0 +1,230 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+
+	"k8s.io/client-go/rest"
+
+	"github.com/golang/glog"
+	"k8s.io/api/admission/v1beta1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+// Event stores the event details
+type Event struct {
```

Comment:
It will be better if we define an interface with the functions exposed outside from event (isMeteringEvent and createMertering) and this structure implements the interface. This way in test cases if we want to mock this interface we can pass a mock implementation of the interface. If you dont want to mock, this is fine.